.github/workflows/all_solutions.yml

```diff
@@ -621,7 +616,6 @@ jobs:
             apt-get update -q -y && apt-get install -q -y curl
             export DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1
             mkdir -p /usr/share/dotnet
-            curl -sSL https://dotnetcli.azureedge.net/dotnet/Sdk/2.1.818/dotnet-sdk-2.1.818-linux-arm64.tar.gz | tar -xzC /usr/share/dotnet
             curl -sSL https://dotnetcli.azureedge.net/dotnet/Sdk/3.1.414/dotnet-sdk-3.1.414-linux-arm64.tar.gz | tar -xzC /usr/share/dotnet
             curl -sSL https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.401/dotnet-sdk-5.0.401-linux-arm64.tar.gz | tar -xzC /usr/share/dotnet
```

Comment:
Do we also want to remove net5 since it is also EOL?

Reply:
I don't think so, since we still target .NET 5 in some test apps, and aren't removing our support for it